proof-systems/src/darlin/marlin/src/test.rs

```diff
@@ -120,7 +120,7 @@ mod marlin {
                 if zk { Some(rng) } else { None },
             );
 
-            assert!(proof.is_ok());
+            // assert!(proof.is_ok());
```

Comment:
Why commented ?

Reply:
Forgot to uncomment after a test. Thanks